### ISSUE TITLE
Just Figure out the files that were changed in last commit and upload those on the server?

### ISSUE THREAD

**Issue by uiyash** [SamKirkland/FTP-Deploy-Action#431]

**Comment by mkursadulusoy** [comment]
I agree with this. I have a project that has too much image in it. Every tim i push something, it uploads every image one by one. I dont need it.

**Comment by Sheldon-Knight** [comment]
> I agree with this. I have a project that has too much image in it. Every tim i push something, it uploads every image one by one. I dont need it.

why dont you exclude them?

**Comment by mkursadulusoy** [comment]
> > I agree with this. I have a project that has too much image in it. Every tim i push something, it uploads every image one by one. I dont need it.
> 
> why dont you exclude them?

Doesn't seem applicable if you have a growing project that has new images.
